### PR TITLE
Check long properties

### PR DESCRIPTION
**Pull request by pascal-brand-st-dev**
This new test is failing when https://github.com/OP-TEE/optee_os/pull/651 is not applied

Signed-off-by: Pascal Brand <pascal.brand@st.com>

### CODE REVIEW

File: ta/os_test/os_test.c [hunk 1]
```diff
@@ -80,17 +80,65 @@ while (true) {
 		     nbuf, (unsigned int)res);
 		return res;
 	}
+	if (vblen != strlen(vbuf) + 1) {
+		EMSG("1TEE_GetPropertyAsString(\"%s\") returned length %u instead of %u\n",
```
Comment (jenswi-linaro) [inline ta/os_test/os_test.c]: Why do you have a number in front of the function name?

Comment (pascal-brand-st-dev) [inline ta/os_test/os_test.c]: I think it was to better identify the trace. Now EMSG dumps the line, it is useless. I'll remove it.

**Comment by pascal-brand-st-dev** [comment]
updated

**Comment by pascal-brand-st-dev** [comment]
Added a test on short properties

**Comment by pascal-brand-st-dev** [comment]
More tests

**Comment by jenswi-linaro** [comment]
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Pull request merged**